coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/PlaybackToSecondary.scala

```diff
@@ -655,7 +655,7 @@ class PlaybackToSecondary[CT, CV](u: PlaybackToSecondary.SuperUniverse[CT, CV],
         return actions
       } catch {
         case e: Throwable =>
-          logger.info("Rolling back to end transaction due to thrown exception: {}", e.getMessage)
+          logger.info("Rolling back to end transaction due to thrown exception: {}", e)
```

Comment:
No placeholder if you want the stack trace (I'm not actually 100% sure what this will do, but I _think_ it'll log `e.toString`).  slf4j treats an extra throwable argument specially.

Reply:
oh okay! thank you!